### PR TITLE
add new rb-1

### PR DESCRIPTION
**Pull request by mandarbapaye**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/mandarbapaye/git-play/pull/7" target="_blank">ReviewNinja

File: ninja1.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+      def reload!
+        reset_memoizations!
+        super
+      end
+
+      protected
+
+        def lookup(locale, key, scope = nil, options = {})
+          flat_key  = I18n::Backend::Flatten.normalize_flat_keys(locale,
```
Comment (mandar-masterclass) [inline ninja1.rb:21]: not good !fix

File: ninja1.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            key, scope, options[:separator]).to_sym
+          flat_hash = memoized_lookup[locale.to_sym]
+          flat_hash.key?(flat_key) ? flat_hash[flat_key] : (flat_hash[flat_key] = super)
+        end
+
+        def memoized_lookup
+          @memoized_lookup ||= Hash.new { |h, k| h[k] = {} }
+        end
+
+        def reset_memoizations!(locale=nil)
```
Comment (mandar-masterclass) [inline ninja1.rb:31]: what is this for !resolve

**Comment by mandar-masterclass** [comment]
General comment @ Conversation

**Pull request closed without merging**